frontend/src/metabase-lib/v1/parameters/utils/targets.unit.spec.ts

```diff
@@ -133,4 +244,330 @@ describe("parameters/utils/targets", () => {
+          const { query, stageIndex, columns } = getParameterColumns(
+            question,
+            parameter,
+          );
+          const columnsInfos = getColumnsInfos(query, stageIndex, columns);
+
+          expect(columnsInfos).toEqual([
+            ["Orders", "Created At"],
+            ["Products", "Product → Created At"],
+          ]);
+        });
+
+        it("date breakouts in multiple stages - returns date column from the last stage only", () => {
+          const question = createQuestion({
+            dataset_query: queryDateBreakoutsMultiStage,
+          });
+          const { query, stageIndex, columns } = getParameterColumns(
+            question,
+            parameter,
+          );
+          const columnsInfos = getColumnsInfos(query, stageIndex, columns);
+
+          expect(columnsInfos).toEqual([["Orders", "Created At: Month"]]);
+        });
+      });
+
+      describe("model", () => {
+        it("no breakouts - returns no columns", () => {
+          const question = createModel({
+            result_metadata: createOrdersTable().fields,
+          });
+          const { columns } = getParameterColumns(question, parameter);
+
+          expect(columns).toHaveLength(0);
+        });
+
+        it("non-date breakout - returns no columns", () => {
+          const question = createModel({
+            dataset_query: queryNonDateBreakout,
+            result_metadata: [createOrdersQuantityField(), createCountField()],
+          });
+          const { columns } = getParameterColumns(question, parameter);
+
+          expect(columns).toHaveLength(0);
+        });
+
+        it("1 date breakout - returns 1 date column", () => {
+          const question = createModel({
+            dataset_query: query1DateBreakout,
+            result_metadata: [createOrdersCreatedAtField(), createCountField()],
+          });
+          const { columns } = getParameterColumns(question, parameter);
+
+          expect(columns).toHaveLength(0);
+        });
+
+        it("2 date breakouts - returns 2 date columns", () => {
+          const question = createModel({
+            dataset_query: query2DateBreakouts,
+            result_metadata: [
+              createOrdersCreatedAtField(),
+              createProductsCreatedAtField(),
+              createCountField(),
+            ],
+          });
+          const { columns } = getParameterColumns(question, parameter);
+
+          expect(columns).toHaveLength(0);
+        });
+
+        it("date breakouts in multiple stages - returns date column from the last stage only", () => {
+          const question = createModel({
+            dataset_query: queryDateBreakoutsMultiStage,
+            result_metadata: [createOrdersCreatedAtField(), createCountField()],
+          });
+          const { columns } = getParameterColumns(question, parameter);
+
+          expect(columns).toHaveLength(0);
+        });
+      });
+    });
+
+    describe("date parameter", () => {
```

Comment:
Just 1 sanity test for any other parameter than unit of time.
Tests for other parameters will come when this behavior changes.